ProQuest.js

```diff
@@ -499,6 +498,33 @@ function getItemType(types) {
 	return guessType;
 }
 
+function scrapeEbraryResults(doc, refs) {
+	// Since we can't chase URLs, let's get what we can from the page
+	var ebraryResults = ZU.xpath(doc, '//div[@class="ebraryitem"]');
```

Comment:
Refactored to do this. Funny enough, as I was doing this, I noticed a hidden span I hadn't noticed before that had a little extra data that wasn't visible on the page (ISBN, language, page count, LoC call number).